### PR TITLE
Handle monitor actions from Redux DevTools

### PR DESCRIPTION
**Pull request by jhen0409**
Currently the `connectReduxDevtools` middleware have no handle monitor actions, in addition of jump state. It will be useful, we can reset state, commit actions, import / export state:

I have yet to handle `TOGGLE_ACTION`, so currently we can't use `SKIP` button.

### CODE REVIEW

File: packages/mst-middlewares/src/redux.ts [hunk 1]
```diff
@@ -58,21 +58,53 @@ export const connectReduxDevtools = function connectReduxDevtools(remoteDevDep:
...
-        if (state) {
-            applyingSnapshot = true
-            mst.applySnapshot(model, state)
-            applyingSnapshot = false
+        if (message.type === "DISPATCH") {
+            handleMonitorActions(remotedev, model, message)
         }
     })
 
+    const initialState = model.toJSON()
```
Comment (mattiamanzati) [inline packages/mst-middlewares/src/redux.ts]: Can you please use "getSnapshot(model)" instead? :)

**Comment by mattiamanzati** [comment]
Please ping me whenever this could be merged :)

**Comment by jhen0409** [comment]
@mattiamanzati it's ready, thanks for review! :D

**Comment by mattiamanzati** [comment]
Awesome! Going to merge! Thanks for your work!

**Pull request merged**